com/TestSources/PyCOMTest/PyCOMTest.idl

```diff
@@ -26,7 +26,7 @@ typedef enum // Missing EnumTestAttributes2
 	uuid(6bcdcb60-5605-11d0-ae5f-cadd4c000000),
 	version(1.1),
 	// an extended character in the help string should stress things...
-	helpstring("Python COM Test Harness 1.0 Type Library, � pywin32 contributors")
+	helpstring("Python COM Test Harness 1.0 Type Library, © pywin32 contributors")
```

Comment:
This is one of only 2 places where this char is used. The other being https://github.com/mhammond/pywin32/blob/c5a30d27cef9e9fe3c4731bd0227fb0060850f40/win32/Lib/win32timezone.py#L10

Everywhere else is either `(c)` (136 times) or `(C)` (17 times)